### ISSUE TITLE
help unclear

### ISSUE THREAD

**Issue by NickyDaB** [xcat2/xcat-core#6083]
I ran `lsdef` with `-h` flag for help.

```
[root@c650mnp02 ~]# lsdef -h

Usage: lsdef - List xCAT data object definitions.

  lsdef [-h | --help ] [-t object-types]

  lsdef [-l|--long] [-a|--all] [-t *object-types*] [-z|--stanza]
        [-i attr-list] [--template [template-object-name]]

  lsdef [-V | --verbose] [-t object-types] [-o object-names]
        [ -l | --long] [-s | --short] [-a | --all] [-z | --stanza ] [-S]
        [-i attr-list] [-w attr==val [-w attr=~val] ...] [noderange]


The following data object types are supported by xCAT:

 auditlog boottarget eventlog firmware group kit kitcomponent kitrepo monitoring network node notification osdistro osdistroupdate osimage pdu policy rack route site taskstate zone zvmivp


Use the '-h' option together with the '-t' option to
 get a list of valid attribute names for each object type.
```

At the very bottom it says:
```
Use the '-h' option together with the '-t' option to
 get a list of valid attribute names for each object type.
```

So I did...

```
[root@c650mnp02 ~]# lsdef -h -t
Error: [c650mnp02]: Invalid option..

Usage: lsdef - List xCAT data object definitions.

  lsdef [-h | --help ] [-t object-types]

  lsdef [-l|--long] [-a|--all] [-t *object-types*] [-z|--stanza]
        [-i attr-list] [--template [template-object-name]]

  lsdef [-V | --verbose] [-t object-types] [-o object-names]
        [ -l | --long] [-s | --short] [-a | --all] [-z | --stanza ] [-S]
        [-i attr-list] [-w attr==val [-w attr=~val] ...] [noderange]


The following data object types are supported by xCAT:

 auditlog boottarget eventlog firmware group kit kitcomponent kitrepo monitoring network node notification osdistro osdistroupdate osimage pdu policy rack route site taskstate zone zvmivp


Use the '-h' option together with the '-t' option to
 get a list of valid attribute names for each object type.

[root@c650mnp02 ~]#
```

But it gave me an error...

```
[root@c650mnp02 ~]# lsdef -h -t
Error: [c650mnp02]: Invalid option..
```

Why?

**Comment by gurevichmark** [comment]
The intentions was to issue something like `lsdef -t node -h`.
But I agree the instructions are not clear.

**Comment by NickyDaB** [comment]
I eventually guessed, the `[-t *object-types*]`   but I had to do some googling to find out all the object types.  

I think at first I was hoping that I could have been told what the list of object types were... 

You can close this issue. but if you want to leave it open because you think  its important  to eventually work on, then you can.

**Comment by gurevichmark** [comment]
Made this and other doc improvements under #6084.
The list of object types is displayed by the `Usage` output:
```
The following data object types are supported by xCAT:

 auditlog boottarget eventlog firmware group kit kitcomponent kitrepo monitoring network node notification osdistro osdistroupdate osimage pdu policy rack route site taskstate zone zvmivp
```

**Comment by NickyDaB** [comment]
I made a group the other day. The group consisted of IB switches. I wasn't sure what objects to make. I went with "node" not sure if that was the best way, but nothing blew up. So I suppose it was ok. 

**My use case was:**
-  Someone told me to make an xCAT group for some switches, so that I could send them a message to update them all at the same time. 
- Don't have experience with xCAT.
- ran the help to get some idea of where to start. 
- after reading, I still didn't feel 100% confident. 
- ended up guessing on what was the way to proceed.
- things "seemed" to work, so I moved on. 

**My expectation:**
- help should have left me feeling confident that I knew what I was doing, and wasn't about to blow up the system. :)
